gc/base/MemoryPoolAddressOrderedList.cpp

```diff
@@ -1842,13 +1914,36 @@ MM_MemoryPoolAddressOrderedList::doFreeEntryAlignmentUpTo(MM_EnvironmentBase *en
 		_darkMatterBytes += lostToAlignment;
 	}
 
-	/* update _firstUnalignedFreeEntry and _prevFirstUnalignedFreeEntry */
-	_firstUnalignedFreeEntry = (NULL == currentFreeEntry) ? FREE_ENTRY_END : currentFreeEntry;
-	if (FREE_ENTRY_END != _firstUnalignedFreeEntry) {
-		_prevFirstUnalignedFreeEntry = (NULL == previousFreeEntry) ? FREE_ENTRY_END : previousFreeEntry;
+	/* update _firstCardUnalignedFreeEntry and _prevCardUnalignedFreeEntry */
+	_firstCardUnalignedFreeEntry = (NULL == currentFreeEntry) ? FREE_ENTRY_END : currentFreeEntry;
+	if (FREE_ENTRY_END != _firstCardUnalignedFreeEntry) {
+		_prevCardUnalignedFreeEntry = (NULL == previousFreeEntry) ? FREE_ENTRY_END : previousFreeEntry;
 	} else {
-		_prevFirstUnalignedFreeEntry = FREE_ENTRY_END;
+		_prevCardUnalignedFreeEntry = FREE_ENTRY_END;
 	}
 
 	return alignedLastFreeEntry;
 }
+
+void
+MM_MemoryPoolAddressOrderedList::setParallelGCAlignment(MM_EnvironmentBase *env, bool satbEnabled)
```

Comment:
formal argument name should be alignementEnabled, free of SATB  knowlwedge (generally who, when, why is doing it) 
the actual argument at the call site is where SATB gets involved, first time
don't forget to update the argument name in the declaration, too